argocd/overlays/moc-infra/projects/chris.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: argoproj.io/v1alpha1
```

Comment:
for the project, 
one of team member is working on this
https://github.com/operate-first/apps/pull/2025/files